docs/examples/multi-debugcallback.c

```diff
@@ -36,7 +36,10 @@
 /* curl stuff */
 #include <curl/curl.h>
 
+#if defined __STDC_VERSION__ && __STDC_VERSION__ > 201710L
```

Comment:
Did you perchance mean `<= 201710L` ? 201710L seems to be the value that [identifies c18](https://github.com/cpredef/predef/blob/master/Standards.md)